test/e2e/kube-ovn/ipam/ipam.go

```diff
@@ -509,4 +509,56 @@ var _ = framework.Describe("[group:ipam]", func() {
 		deploy = deployClient.PatchSync(deploy, patchedDeploy)
 		checkFn()
 	})
+
+	framework.ConformanceIt("should allocate right IPs for the statefulset when there are multiple IP Pools added to its namespace", func() {
```

Comment:
Let's skip this test case for versions <= v1.14 by using `f.SkipVersionPriorTo(1, 14, "some message...")`